.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-merge-conflict
+  - repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
+    hooks:
+      - id: git-secrets
+        entry: "git-secrets --scan"
+        files: "."
+  - repo: https://github.com/aws-cloudformation/cfn-python-lint
```

Comment:
What percentage of repos in this org have CloudFormation templates? This doesn't strike me as something that should be in *every* repo.

Reply:
I would guess almost all repos contain CloudFormation and this is non-harmful to repos that don't contain cfn anyway as it only looks for files with a `.template` extension :)

Reply:
Not the exact number we're looking for but 245 files in the org are suffixed with `.template`, 445 files named `template.yaml`. Not sure on the best way to try and find the % of repos in this org with CFN templates as my GitHub search skillz have failed me